tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -2089,27 +2089,27 @@ Count_Person_BlackOrAfricanAmericanAlone_ResidesInNursingFacilities,Number of Bl
 Count_Person_BornInOtherStateInTheUnitedStates,Number of people who were born in other state in the United States
 Count_Person_BornInStateOfResidence,Number of people who were born in state of residence
 Count_Person_Civilian_Employed,Number of employed civilian
-Count_Person_Civilian_FamilyHousehold_NonInstitutionalized,Number of people in family household
-Count_Person_Civilian_Female_NonInstitutionalized,Number of NonInstitutionalized females
-Count_Person_Civilian_InLaborForce,Number of people in labor force
-Count_Person_Civilian_Male_NonInstitutionalized,Number of NonInstitutionalized males
-Count_Person_Civilian_MarriedCoupleFamilyHousehold_NonInstitutionalized,Number of people in married couple family household
-Count_Person_Civilian_NoDisability_NonInstitutionalized,Number of NonInstitutionalized females without disability
-Count_Person_Civilian_NonInstitutionalized,Number of NonInstitutionalized civilian
-Count_Person_Civilian_NonInstitutionalized_AmericanIndianOrAlaskaNativeAlone,number of NonInstitutionalized American Indian or Alaska Native Civilian
-Count_Person_Civilian_NonInstitutionalized_AsianAlone,number of NonInstitutionalized asian Civilian
-Count_Person_Civilian_NonInstitutionalized_BlackOrAfricanAmericanAlone,number of NonInstitutionalized Black or African American Civilian
-Count_Person_Civilian_NonInstitutionalized_ForeignBorn,number of NonInstitutionalized Foreign Born Civilian
-Count_Person_Civilian_NonInstitutionalized_HispanicOrLatino,number of NonInstitutionalized Hispanic or Latino Civilian
-Count_Person_Civilian_NonInstitutionalized_Native,number of NonInstitutionalized native Civilian
-Count_Person_Civilian_NonInstitutionalized_NativeHawaiianOrOtherPacificIslanderAlone,number of NonInstitutionalized Native Hawaiian or Other Pacific Islander Civilian
-Count_Person_Civilian_NonInstitutionalized_OneRace,number of NonInstitutionalized single race Civilian
-Count_Person_Civilian_NonInstitutionalized_PovertyStatusDetermined,number of NonInstitutionalized Civilian with poverty status determined
-Count_Person_Civilian_NonInstitutionalized_ResidesInHousehold,number of NonInstitutionalized Civilian who resides in household
-Count_Person_Civilian_NonInstitutionalized_TwoOrMoreRaces,number of NonInstitutionalized muti-race Civilian
-Count_Person_Civilian_NonInstitutionalized_WhiteAlone,number of NonInstitutionalized white Civilian
+Count_Person_Civilian_FamilyHousehold_NonInstitutionalized,Number of civilian in family household

```

Comment:
plural - civilians

Reply:
done

svindex diffs: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_10_16_28_40.html
- no new diffs that are concerning